LondonBritishLibrary/orient/BLorient8814.xml

```diff
@@ -481,13 +476,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <origin>
                             <origDate notBefore="1700" notAfter="1799">18th century</origDate>
                         </origin>
-                        <provenance>According to <bibl><ptr target="bm:Emmel2023Curzon"/><citedRange unit="page">236-241</citedRange>
-                            </bibl> acquired by <persName ref="PRS3244Curzon"/> during his stay in Egypt in 
-                            <date notBefore="1837" notAfter="1838">1837-1838</date>. Three years after his death on 
-                            <date when="1876-04-19">19 April 1876</date>the codex was given as a deposit on loan to the British Museum 
-                            (later: <placeName ref="INS00001BL">British Library</placeName>), then as a bequest from his daughter 
-                            <persName ref="PRS14320DareaCurzon">Darea</persName> in <date when="1917-10-13">13 October 1917</date> 
-                            as inscribed in a note on the front flyleaf.</provenance>
+                        <provenance>The first owner was <persName role="owner" ref="PRS14385WaldaMadhen"/>. His name is inscribed on 
+                            several folios as for instance on <locus target="#65rb"/>.
+                            According to <bibl><ptr target="bm:Emmel2023Curzon"/><citedRange unit="page">236-241</citedRange></bibl> 
+                            the manuscript was acquired by <persName ref="PRS3244Curzon"/> during his stay in Egypt in 
+                            <date notBefore="1837" notAfter="1838">1837-1838</date>. 
+                            Three years after his death on <date when="1876-04-19">19 April 1876</date>the codex was given as a deposit on loan to 
```

Comment:
The history of the acquisition of the ms by British Museum should be placed in <acquisition/>, part in <history/>

Reply:
The history of the acquisition of the ms by British Museum should be placed in `<acquisition/>` , a part under `<history/>`